src/Analyzers/CSharp/CodeFixes/UseObjectInitializer/CSharpUseObjectInitializerCodeFixProvider.cs

```diff
@@ -59,9 +60,12 @@ public CSharpUseObjectInitializerCodeFixProvider()
                 var match = matches[i];
                 var expressionStatement = match.Statement;
                 var assignment = (AssignmentExpressionSyntax)expressionStatement.Expression;
+                var trivia = match.MemberAccessExpression.GetLeadingTrivia();
+
+                var newTrivia = i == 0 ? trivia.WithoutLeadingBlankLines() : trivia;
```

Comment:
do we need the same fix for:

1. VB?
2. the 'use collection initializer' equivalent feature?

Reply:
1. No, because comments are not allowed inside initializers in VB:
![devenv_BbPcEVF5K9](https://user-images.githubusercontent.com/70431552/157096248-ddb93d21-dd4d-431f-8e8e-57cfb06f8dfc.gif)
2. Yes we do, nice reminder!